libcontainer/cgroups/fs2/fs2.go

```diff
@@ -141,6 +141,12 @@ func (m *manager) GetStats() (*cgroups.Stats, error) {
 			errs = append(errs, err)
 		}
 	}
+	// rdma (since kernel 4.11)
+	if _, ok := m.controllers["rdma"]; ok {
```

Comment:
Since commit 0f8d2b6b we no longer check if the controller is available.

Reply:
Sure i'll do the required changes.

Reply:
This ^^^ is still a valid concern.

Reply:
Resolved in latest commit.